### ISSUE TITLE
detect method

### ISSUE THREAD

**Issue by drSun** [wfstudioru/yii2-yandex-translate-api#1]
Привет! Спасибо за полезное расширение :) не сможешь ли добавить какой-нибудь метод для совершения произвольных запросов или просто запроса определения языка? Описание тут https://tech.yandex.ru/translate/doc/dg/reference/detect-docpage/
Возможно, было бы хорошо сделать метод getResponse более универсальным, чтобы можно было определять язык или получать список языков тоже.

**Comment by BaBL86** [comment]
Вечер добрый. Не совсем понял? Вам нужен метов, чтобы передать текст и
получить язык?

25 ноября 2015 г., 21:07 пользователь Roman notifications@github.com
написал:

**Comment by drSun** [comment]
Да, верно, или я ошибаюсь и можно уже существующими методами такое сделать?
